### PR TITLE
replace nginx related comment and variable names.

### PR DESCRIPTION
**Pull request by secwang**
replace nginx related comment and variable names from sample-bt XD.

### CODE REVIEW

File: sample-bt [hunk 1]
```diff
@@ -71,11 +71,11 @@ if ($^O ne 'linux') {
 
 my $exec_file = "/proc/$pid/exe";
 if (!-f $exec_file) {
-    die "Nginx process $pid is not running or ",
+    die "Process $pid is not running or ",
         "you do not have enough permissions.\n";
-}
```
Comment (agentzh) [inline sample-bt]: Why removing this? It makes the Perl script have syntax error....

Comment (agentzh) [inline sample-bt]: @secwang I think you should at least run the modified script yourself before committing ;)

**Comment by agentzh** [comment]
Merged with the syntax error fix. Thanks for your contribution!

**Comment by secwang** [comment]
sorry , will test next time

**Pull request closed without merging**